Code/GraphMol/Chirality.cpp

```diff
@@ -402,9 +402,17 @@ void findAtomNeighborsHelper(const ROMol &mol, const Atom *atom,
   }
 }
 
+// conditions for an atom to be a candidate for ring stereochem:
+//   1) two non-ring neighbors that have different ranks
+//   2) one non-ring neighbor and two ring neighbors (the ring neighbors will
+//      have the same rank)
+//   3) four ring neighbors with three different ranks
```

Comment:
pedantic:  with at least three different ranks (and with at least two different ranks below)